ocaml/xapi/xapi_globs.ml

```diff
@@ -915,6 +915,10 @@ let winbind_cache_time = ref 60
 
 let winbind_machine_pwd_timeout = ref (7 * 24 * 3600)
 
+let winbind_update_closest_kdc_interval = ref (3600. *. 24.) (* every day *)
+
+let winbind_update_closest_kdc_delay = ref 5. (* Delay 5 seconds to make way for xapi startup *)
```

Comment:
this is not used. It's also probably not a good idea to try and guess an upper bound for how long it takes for xapi to start

Reply:
my bad, I comment it out for debug.

My understand about it is the delay to execute the function after push to queue. 
We need a value for this during xapi startup, to avoid take too many resources from xapi.
Or we can just use 0. if no concern.

Reply:
fair enough. As I understand it, the delay time before trying to find the closest kdc is really 5s + the amount of time it takes to check whether winbind is ready, but I assume the latter does not really take that long in the average case

fwiw, it may also be simpler to just put these constants in the winbind ml file, but I am not too strongly opinionated on that

Reply:
I put it into xapi_glob.ml to make it configurable.